### PR TITLE
Add a flag to disable the activationid

### PR DESCRIPTION
**Pull request by bigmianhua**
Hi @jthomas, just updated the code, changes made to util.js and index.js, let me know if you have any question about it, thank you so much

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -34,7 +34,10 @@ const metrics = fn => {
 
 // converts metric name into fully qualified label
 metrics.label = util.label
-  
+
+// default config
+metrics.config = [];
```
Comment (jthomas) [inline index.js]: Should be an object and not an array?

**Review by jthomas — CHANGES_REQUESTED** [review]

**Comment by jthomas** [comment]
Code LGTM (apart from that tiny change). 

Couple of things needed before we can merge.... 

- [ ] Add new tests for the `metrics/util.js` code to [here](https://github.com/jthomas/openwhisk-metrics/blob/master/test/metrics.js).
- [x] Update [documentation](https://github.com/jthomas/openwhisk-metrics#configuration), add new section at the bottom of this configuration section.

File: test/metrics.js [hunk 2]
```diff
@@ -74,3 +75,42 @@ test('cold start metric should return bool for first invocations', t => {
...
+})
+
+test('util.label function should return label name without activation id for the metrics when config ignore_activation_ids parameter is set to true', t => {
+  process.env['__OW_NAMESPACE'] = 'testNamespace';
+  process.env['__OW_ACTION_NAME'] = '/testNamespace/testName';
+  process.env['__OW_ACTIVATION_ID']= 'testActivationId';
+  config = {};
+  config.ignore_activation_ids = true;
+  let result = util.label('testLabel', config);
+  t.is(result, 'testNamespace.testName.testActivationId.testLabel');
```
Comment (jthomas) [inline test/metrics.js]: Should this be 'testNamespace.testName.testLabel'?

**Comment by jthomas** [comment]
Are the tests finished and ready for review? I've tried it locally but they aren't working atm.

**Comment by bigmianhua** [comment]
@jthomas sorry, I was out few days ago, I have added a test code in test/metric.js to test util.label function based on config. Please let me know if I need to make any change on it. Thank you so much

**Comment by jthomas** [comment]
@bigmianhua Thanks for adding the tests. Travis build is failing, can you investigate and resolve?

**Comment by bigmianhua** [comment]
@jthomas Hi James, we have pushed the test code, and looks like it passes the test, let me know if there have any other issues, thank you!

**Comment by jthomas** [comment]
Awesome. Thanks for the contribution. 💪💪💪
I've published the new library version to NPM. Thanks for the help with this feature.

**Pull request merged**